cmd/docker/docker_test.go

```diff
@@ -26,24 +25,25 @@ func TestClientDebugEnabled(t *testing.T) {
 }
 
 func TestExitStatusForInvalidSubcommandWithHelpFlag(t *testing.T) {
-	discard := ioutil.Discard
-	cmd := newDockerCommand(command.NewDockerCli(os.Stdin, discard, discard, false, nil))
+	cmd := newDockerCommand(command.NewDockerCli(command.WithInputStream(os.Stdin)))
```

Comment:
cf what I wrote above this appears to rely on the streams defaulting to discard.